### ISSUE TITLE
Module not found: 'cypress/types/lodash'

### ISSUE THREAD

**Issue by alechuang98** [ntuee-webprogramming/hack1-QAs#23]
yarn start後遇到以下Error
yarn install是好的

```
Failed to compile.

Error in ./src/util/createBoard.js
Module not found: 'cypress/types/lodash' in <userPath>/wp1101/hack1/src/util

 @ ./src/util/createBoard.js 7:14-45
```

**Comment by peterhuangfu** [comment]
請參考 closed #15 issue

**Comment by chinyi0523** [comment]
Please refer to #15
